packages/bot-skeleton/src/scratch/help-content/help-strings/controls_forEach.js

```diff
@@ -9,6 +9,6 @@ export default {
         localize(
             'In this example, the loop will repeat three times, as that is the number of items in the given list. During each iteration, the variable "i" will be assigned a value from the list. '
         ),
-        localize('Please note that changing the value of "i" won\'t change the value of the original item in the list'),
+        localize(`Please note that changing the value of "i" won't change the value of the original item in the list`),
```

Comment:
why not change the translation script so it understands `\` instead? I think we won't remember to use back tick for any strings with single quote in them @aaron-binary @oskar-binary

Reply:
That's a good idea. 👍